microsoft-edge/devtools-guide-chromium/evaluate-performance/unminify.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Unminify performance profiles
+description: Tutorial for unminifying the call stack in a performance profile using source maps.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.date: 03/08/2022
+---
+# Unminify performance profiles
+
+<!-- todo: review all mentions of "source mapping" in this article - what is the relation between these 3 features/technologies?
```

Comment:
@captainbrosset - could you pls check how we present the benefits/UX of these 3 distinct technologies/techniques?

*  source mapping and its UX/benefits
*  unminification and its UX/benefits
*  ADO secure server (for source maps) and its UX/benefits

Reply:
The thing is that these are not 3 distinct technologies. Your first bullet point is the technology that devtools uses in several places. The second point is the act of transforming code in various panels of devtools using source maps. And the third is the same thing, but also using ADO to help with security.

So I don't think it makes sense to create a table that explains what each of these are and what their benefits are. They're all linked. You can't unminify code without source maps. You can't use ADO's secure server without source maps.

Now, there's a bit of possible confusion with the word "unminify". It also sometimes refers to prettifying code in the Sources tool. When code is not nicely formatted (often all on 1 line), there's a pretty-print button at the bottom people can use to reformat the code. This feature does not use source maps behind the scenes. It parses the code to understand its syntax and produces a nicely formatted version based on this. 
I know I used to call this feature minification, and I think other people might too. Devtools now calls this "pretty printing".

What the performance tool does, with the "unminify profiles" feature is NOT pretty-printing. It uses source maps to swap production function names with original function names in perf profiles.

I think this article would benefit from removing the word "unminify", and replace this with something like "map to original function names".

2 problems with this: 
1. it's much longer
2. the devtools UI already has the word "unminify"

But, I don't think we should write a table with the UX/benefits of the 3 technologies you mentioned. 
I think we should a really good intro to source maps and how devtools uses them throughout the tools for better developer UX. And then refer to this intro in other articles, like this one.